packages/block-editor/src/hooks/block-bindings.js

```diff
@@ -286,7 +231,7 @@ export const BlockBindingsPanel = ( { name, metadata } ) => {
 				} );
 				// Only add source if the list is not empty.
 				if ( sourceList ) {
-					fieldsList[ label ] = { ...sourceList };
+					fieldsList[ sourceName ] = { ...sourceList };
```

Comment:
It seems that we were just using the values of the `registeredSources` object instead of their keys.

I'm not sure if if sources label were always the same as the registeredSources key, but this approach seems safer, as the label should allow whitespaces or uppercases.

Reply:
> Why were we using the label here before?

I believe the name wasn't exposed before. But this seemed safer to me as well.